internal/business/persisted_operations/persisted_operations.go

```diff
@@ -35,6 +36,14 @@ var (
 		ConstLabels: nil,
 	},
 		[]string{"system", "result"})
+	reloadGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace:   "graphql_protect",
+		Subsystem:   "persisted_operations",
+		Name:        "GCS_download_speed",
```

Comment:
```suggestion
		Name:        "gcs_download_speed",
```

Im wondering if `speed` is the right terminology here. I'd expect a measure of Kb/s or Mb/s while I'm guessing you're trying to track duration?

Reply:
good one